### PR TITLE
PT-2103: Merge Templates and Import functions on Pedigree.

### PR DESCRIPTION
**Pull request by veronikaslc**
Fixed.

### CODE REVIEW

**Comment by veronikaslc** [comment]
@allasm  @antonkatz please review

File: components/pedigree/resources/src/main/resources/pedigree/saveLoadEngine.js [hunk 1]
```diff
@@ -193,7 +193,8 @@ define([
                         // since we just loaded data from disk data in memory is equivalent to data on disk
                         editor.getUndoRedoManager().addSaveEvent();
                     } else {
-                        new TemplateSelector(true);
+                        document.fire("pedigree:load:finish");
+                        $('action-templates').click();
```
Comment (antonkatz) [inline components/pedigree/resources/src/main/resources/pedigree/saveLoadEngine.js]: What does this do? Should it not call some sort of a function directly?

File: components/pedigree/resources/src/main/resources/pedigree/view/templateSelector.js [hunk 2]
```diff
@@ -92,8 +76,7 @@ define([
          * @private
          */
         _onTemplateSelected: function(event, pictureBox) {
-            //console.log("observe onTemplateSelected");
-            this.dialog.close();
+            $$('.pedigree-import-chooser .msdialog-close')[0].click();
```
Comment (antonkatz) [inline components/pedigree/resources/src/main/resources/pedigree/view/templateSelector.js]: If ```.pedigree-import-chooser``` is a dialog then you should use its ```close()``` method.

File: components/pedigree/resources/src/main/resources/pedigree/view/importSelector.js [hunk 3]
```diff
@@ -165,7 +163,7 @@ define([], function(){
             var importValue = this.importValue.value;
             console.log("Importing:\n" + importValue);
             
-            this.hide();
+            $$('.pedigree-import-chooser .msdialog-close')[0].click();
```
Comment (antonkatz) [inline components/pedigree/resources/src/main/resources/pedigree/view/importSelector.js]: see below

File: components/pedigree/resources/src/main/resources/pedigree/pedigree.js [hunk 4]
```diff
@@ -148,14 +148,51 @@ define([
...
+            var tabs = [];
+            var tabHeaders = [];
+            var tabTop = new Element('dl', {'class':'template-import-tabs'});
+            tabs.push(editor.getTemplateSelector().mainDiv);
+            tabs.push(editor.getImportSelector().mainDiv);
+            tabHeaders.push(new Element('dd', {"class": "active"}).insert("<a>" + "Pedigree templates" + "</a>"));
+            tabHeaders.push(new Element('dd').insert("<a>" + "Pedigree import" + "</a>"));
+            var switchTab = function(index) {
+                return function() {
+					var isActive = tabHeaders[index].hasClassName('active');
```
Comment (antonkatz) [inline components/pedigree/resources/src/main/resources/pedigree/pedigree.js]: The formatting here is wayy to the right

File: components/pedigree/resources/src/main/resources/pedigree/saveLoadEngine.js (hunk 1 above)
Comment (veronikaslc) [inline components/pedigree/resources/src/main/resources/pedigree/saveLoadEngine.js]: It basically opens the dialog with tabs with some additional settings, a listener is attached to a button and its function has some lines of code, pedegree.js lines 189-193.

File: components/pedigree/resources/src/main/resources/pedigree/view/templateSelector.js (hunk 2 above)
Comment (veronikaslc) [inline components/pedigree/resources/src/main/resources/pedigree/view/templateSelector.js]: I don't have the dialogModal class object here to use its close() method If I use TemplateSelector class inside the PedigreeEditor class before I create dialogModal in it.

**Comment by allasm** [comment]
cherry-picked into PT-1548 branch

**Pull request closed without merging**